pkg/asset/installconfig/aws/validation.go

```diff
@@ -4,22 +4,26 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws/awserr"
```

Comment:
Please move to correct place in the ordered list of imports.

Reply:
done